pkg/serverless/invocationlifecycle/lifecycle.go

```diff
@@ -20,15 +23,22 @@ type LifecycleProcessor struct {
 	ProcessTrace        func(p *api.Payload)
 	Demux               aggregator.Demultiplexer
 	DetectLambdaLibrary func() bool
+	ExecutionContext    *serverlessLog.ExecutionContext
 }
 
 // OnInvokeStart is the hook triggered when an invocation has started
-func (lp *LifecycleProcessor) OnInvokeStart(startDetails *InvocationStartDetails) {
+func (lp *LifecycleProcessor) OnInvokeStart(startDetails *InvocationStartDetails, ctx *serverlessLog.ExecutionContext) {
 	log.Debug("[lifecycle] onInvokeStart ------")
 	log.Debug("[lifecycle] Invocation has started at :", startDetails.StartTime)
 	log.Debug("[lifecycle] Invocation invokeEvent payload is :", startDetails.InvokeEventRawPayload)
 	log.Debug("[lifecycle] ---------------------------------------")
 
+	if strings.ToLower(os.Getenv("DD_TRACE_ENABLED")) == "true" &&
```

Comment:
[strconv.ParseBool](https://pkg.go.dev/strconv#ParseBool) might be a better alternative here